packages/editor/src/components/post-featured-image/style.scss

```diff
@@ -39,22 +31,31 @@
 	box-shadow: 0 0 0 0 var(--wp-admin-theme-color);
 	overflow: hidden; // Ensure the focus style properly encapsulates the image.
 
+	// Apply a max-height.
 	display: flex;
 	justify-content: center;
-	height: 150px;
+	max-height: 150px;
```

Comment:
Do we need a `max-height`? I think this is effectively handled by the `aspect-ratio` on the image.

Reply:
I think not. I didn't observe anything weird without it and tested with various image(tall ones, small ones, etc..). I removed it.